crates/bevy_render/src/render_resource/storage_buffer.rs

```diff
@@ -78,6 +98,21 @@ impl<T: ShaderType + WriteInto> StorageBuffer<T> {
     }
 }
 
+/// Stores data to be transferred to the GPU and made accessible to shaders as a dynamic storage buffer.
+///
+/// Dynamic storage buffers can be made available to shaders as some combination of read/write, unlike
+/// [`UniformBuffer`](crate::render_resource::UniformBuffer) which is read-only. Furthermore, dynamic storage buffers
+/// can store much larger data than uniform buffers, which are best suited to relatively small data. Dynamic storage buffers
+/// are particularly well-suited to storing like a Rust-vector (wgpu's "runtime-sized arrays") and have a
+/// [`push`](crate::render_resource::DynamicStorageBuffer::push) method, unlike
+/// [`StorageBuffer`](crate::render_resource::StorageBuffer).
```

Comment:
I don’t agree with this. I’ll try to make a suggestion soon.

Reply:
I'd probably rather say that StorageBuffer is well-suited to storing a single runtime-sized array like a Rust Vec, unlike UniformBuffer which are limited size (minimum guaranteed size is 16kB) and can only contain fixed-size arrays.